### PR TITLE
Update install-a-delegate-with-3-rd-party-tool-custom-binaries.md

### PR DESCRIPTION
**Pull request by kat-enos**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)* Finishes the division of one page into two by adding a link between them. Adds some edits to boot now that the larger format task is done.

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/platform/2_Delegates/advanced-installation/install-a-delegate-with-3-rd-party-tool-custom-binaries.md [hunk 1]
```diff
@@ -1,61 +1,54 @@
...
 ---
 
-Harness delegates include binaries for the third-party SDKs that are required for Harness-supported integrations including Kubernetes, Helm, and so on. 
+Delegates are packaged with third-party SDKs that support Kubernetes, Helm, and other Harness-integrated tools. The SDKs are included on the delegate image as binary files; depending on the tool, multiple versions are included. 
 
-Harness includes multiple binary versions to support customers using code that requires versions other than the latest.
+Harness provides a "minimal" delgate image that does not include third-party SDKs. You can use YAML to configure the minimal delegate image for the installation of the tools and versions you select. You can install software on the delegate using the `INIT_SCRIPT` environment variable.
 
-For a list of the SDK versions that are certified for different types of deployments, see Delegate-required SDKs.
+For a list of the SDK versions that are certified for different types of deployments, see [Delegate-required SDKs](/docs/platform/2_Delegates/delegate-reference/delegate-required-sdks.md).
```
Comment (SudheendraKatte) [inline docs/platform/2_Delegates/advanced-installation/install-a-delegate-with-3-rd-party-tool-custom-binaries.md:15]: Style suggestion - ...go to [Delegate-required SDKs](/docs...

**Review by SudheendraKatte — APPROVED** [review]
Hi @kat-enos , approved with a minor suggestion.

File: docs/platform/2_Delegates/advanced-installation/install-a-delegate-with-3-rd-party-tool-custom-binaries.md (hunk 1 above)
Comment (kat-enos) [reply docs/platform/2_Delegates/advanced-installation/install-a-delegate-with-3-rd-party-tool-custom-binaries.md:15]: @SudheendraKatte I added it although I think that change was generally intended to be implemented in the docs gradually, as new content was added. Please approve a second time. Thanks.

**Review by SudheendraKatte — APPROVED** [review]
LGTM

**Comment by kat-enos** [comment]
Thanks @SudheendraKatte.

**Pull request merged**